Scripts/assignment/freight_assignment.py

```diff
@@ -44,12 +44,26 @@ def save_network_volumes(self, commodity_class: str):
         for ass_class in param.freight_modes:
             spec = self._freight_specs[ass_class].ntw_results_spec
             attr_name = (commodity_class + ass_class)[:17]
-            spec["on_segments"]["transit_volumes"] = '@' + attr_name
-            spec["on_links"]["aux_transit_volumes"] = '@a_' + attr_name
+            spec["on_segments"]["transit_volumes"] = "@" + attr_name
+            spec["on_links"]["aux_transit_volumes"] = "@a_" + attr_name
             self.emme_project.network_results(
                 spec, self.emme_scenario, ass_class)
+        spec = self._car_spec.truck_spec()
+        attr_name = (commodity_class + "truck")[:17]
+        spec["classes"][0]["results"]["link_volumes"] = "@" + attr_name
```

Comment:
Commodity specific truck assignment was missing. Added into this development phase.